diff -rupN a/gtk/Makefile.am b/gtk/Makefile.am
--- a/gtk/Makefile.am	2017-11-12 05:00:27.000000000 +0100
+++ b/gtk/Makefile.am	2017-11-14 01:03:32.897568821 +0100
@@ -611,10 +611,10 @@ install-data-local:
 if ENABLE_GTK_LEGACY
 	$(MKDIR_P) $(adaptadir)/gtk-3.0
 	cp -Rv $(gtk318_file) $(adaptadir)/gtk-3.0
-	cp -Rv $(gtk3_thumb_file) $(adaptadir)/gtk-3.0
+#	cp -Rv $(gtk3_thumb_file) $(adaptadir)/gtk-3.0
 	$(MKDIR_P) $(noktodir)/gtk-3.0
 	cp -Rv $(gtk318nokto_file) $(noktodir)/gtk-3.0
-	cp -Rv $(gtk3_darkthumb_file) $(noktodir)/gtk-3.0/thumbnail.png
+#	cp -Rv $(gtk3_darkthumb_file) $(noktodir)/gtk-3.0/thumbnail.png
 
 	$(MKDIR_P) $(adaptadir)/gtk-3.20
 	cp -Rv $(gtk320_file) $(adaptadir)/gtk-3.20
@@ -627,16 +627,16 @@ if ENABLE_GTK_LEGACY
 else
 	$(MKDIR_P) $(adaptadir)/gtk-3.0
 	cp -Rv $(gtk320_file) $(adaptadir)/gtk-3.0
-	cp -Rv $(gtk3_thumb_file) $(adaptadir)/gtk-3.0
+#	cp -Rv $(gtk3_thumb_file) $(adaptadir)/gtk-3.0
 	$(MKDIR_P) $(adaptaetadir)/gtk-3.0
 	cp -Rv $(gtk320eta_file) $(adaptaetadir)/gtk-3.0
-	cp -Rv $(gtk3_thumb_file) $(adaptaetadir)/gtk-3.0
+#	cp -Rv $(gtk3_thumb_file) $(adaptaetadir)/gtk-3.0
 	$(MKDIR_P) $(noktodir)/gtk-3.0
 	cp -Rv $(gtk320nokto_file) $(noktodir)/gtk-3.0
-	cp -Rv $(gtk3_darkthumb_file) $(noktodir)/gtk-3.0/thumbnail.png
+#	cp -Rv $(gtk3_darkthumb_file) $(noktodir)/gtk-3.0/thumbnail.png
 	$(MKDIR_P) $(noktoetadir)/gtk-3.0
 	cp -Rv $(gtk320noktoeta_file) $(noktoetadir)/gtk-3.0
-	cp -Rv $(gtk3_darkthumb_file) $(noktoetadir)/gtk-3.0/thumbnail.png
+#	cp -Rv $(gtk3_darkthumb_file) $(noktoetadir)/gtk-3.0/thumbnail.png
 endif
 	$(MKDIR_P) $(adaptadir)/gtk-3.22
 	cp -Rv $(gtk322_file) $(adaptadir)/gtk-3.22
@@ -778,16 +778,16 @@ endif
 if ENABLE_GTK_NEXT
 	$(MKDIR_P) $(adaptadir)/gtk-4.0
 	cp -Rv $(gtknext_file) $(adaptadir)/gtk-4.0
-	cp -Rv $(gtk3_thumb_file) $(adaptadir)/gtk-4.0
+#	cp -Rv $(gtk3_thumb_file) $(adaptadir)/gtk-4.0
 	$(MKDIR_P) $(adaptaetadir)/gtk-4.0
 	cp -Rv $(gtknexteta_file) $(adaptaetadir)/gtk-4.0
-	cp -Rv $(gtk3_thumb_file) $(adaptaetadir)/gtk-4.0
+#	cp -Rv $(gtk3_thumb_file) $(adaptaetadir)/gtk-4.0
 	$(MKDIR_P) $(noktodir)/gtk-4.0
 	cp -Rv $(gtknextnokto_file) $(noktodir)/gtk-4.0
-	cp -Rv $(gtk3_darkthumb_file) $(noktodir)/gtk-4.0/thumbnail.png
+#	cp -Rv $(gtk3_darkthumb_file) $(noktodir)/gtk-4.0/thumbnail.png
 	$(MKDIR_P) $(noktoetadir)/gtk-4.0
 	cp -Rv $(gtknextnoktoeta_file) $(noktoetadir)/gtk-4.0
-	cp -Rv $(gtk3_darkthumb_file) $(noktoetadir)/gtk-4.0/thumbnail.png
+#	cp -Rv $(gtk3_darkthumb_file) $(noktoetadir)/gtk-4.0/thumbnail.png
 endif
 
 if ENABLE_XFCE
diff -rupN a/shell/Makefile.am b/shell/Makefile.am
--- a/shell/Makefile.am	2017-11-12 05:00:27.000000000 +0100
+++ b/shell/Makefile.am	2017-11-14 00:11:25.275489170 +0100
@@ -276,8 +276,8 @@ cinnamon_asset_panel_file = \
 cinnamon_asset_switch_file = \
 	$(cinnamondir)/assets/switch/switch-off.svg \
 	$(cinnamondir)/assets/switch/switch-on.svg
-cinnamon_asset_thumb_file = \
-	$(cinnamondir)/assets/thumbnail.png
+#cinnamon_asset_thumb_file = \
+#	$(cinnamondir)/assets/thumbnail.png
 
 cinnamonnoktodir    = $(srcdir)/cinnamon-nokto
 cinnamon_nokto_file = \
@@ -317,8 +317,8 @@ cinnamon_nokto_asset_panel_file = \
 cinnamon_nokto_asset_switch_file = \
 	$(cinnamonnoktodir)/assets/switch/switch-off.svg \
 	$(cinnamonnoktodir)/assets/switch/switch-on.svg
-cinnamon_nokto_asset_thumb_file = \
-	$(cinnamonnoktodir)/assets/thumbnail.png
+#cinnamon_nokto_asset_thumb_file = \
+#	$(cinnamonnoktodir)/assets/thumbnail.png
 
 unitydir   = $(srcdir)/unity
 unity_file = \
@@ -490,7 +490,7 @@ if ENABLE_CINNAMON
 	$(MKDIR_P) $(srcdir)/cinnamon-nokto/assets/panel
 	$(MKDIR_P) $(srcdir)/cinnamon-nokto/assets/switch
 	cd $(srcdir)/asset && ./clone-assets-cinnamon.sh
-	cd $(srcdir)/asset && ./render-assets-cinnamon-thumb.sh
+#	cd $(srcdir)/asset && ./render-assets-cinnamon-thumb.sh
 endif
 
 if ENABLE_UNITY
@@ -612,10 +612,10 @@ if ENABLE_CINNAMON
 		$(adaptadir)/cinnamon/assets/panel
 	cp -Rv $(cinnamon_asset_switch_file) \
 		$(adaptadir)/cinnamon/assets/switch
-	cp -Rv $(cinnamon_asset_thumb_file) \
-		$(adaptadir)/cinnamon/assets
-	cd $(adaptadir)/cinnamon && \
-		ln -sf ./assets/thumbnail.png thumbnail.png
+#	cp -Rv $(cinnamon_asset_thumb_file) \
+#		$(adaptadir)/cinnamon/assets
+#	cd $(adaptadir)/cinnamon && \
+#		ln -sf ./assets/thumbnail.png thumbnail.png
 
 	$(MKDIR_P) $(noktodir)/cinnamon
 	$(MKDIR_P) $(noktodir)/cinnamon/assets
@@ -635,10 +635,10 @@ if ENABLE_CINNAMON
 		$(noktodir)/cinnamon/assets/panel
 	cp -Rv $(cinnamon_nokto_asset_switch_file) \
 		$(noktodir)/cinnamon/assets/switch
-	cp -Rv $(cinnamon_nokto_asset_thumb_file) \
-		$(noktodir)/cinnamon/assets
-	cd $(noktodir)/cinnamon && \
-		ln -sf ./assets/thumbnail.png thumbnail.png
+#	cp -Rv $(cinnamon_nokto_asset_thumb_file) \
+#		$(noktodir)/cinnamon/assets
+#	cd $(noktodir)/cinnamon && \
+#		ln -sf ./assets/thumbnail.png thumbnail.png
 endif
 
 if ENABLE_UNITY
@@ -751,8 +751,6 @@ if ENABLE_CINNAMON
 CLEANFILES += \
 	$(srcdir)/cinnamon/cinnamon.css \
 	$(srcdir)/cinnamon-nokto/cinnamon.css \
-	$(srcdir)/cinnamon/assets/thumbnail.png \
-	$(srcdir)/cinnamon-nokto/assets/thumbnail.png \
 	$(srcdir)/cinnamon/assets/*/*.svg \
 	$(srcdir)/cinnamon-nokto/assets/*/*.svg \
 	$(srcdir)/asset/assets-cinnamon/common/checkbox/*.svg \
