From 77d1928e70105a3b747333b7386d7951df306301 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mateusz=20Miku=C5=82a?= <mati865@gmail.com>
Date: Wed, 17 Jan 2018 00:25:05 +0100
Subject: [PATCH] Use Arch mount dir on Manjaro

---
 dirs/dirs.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dirs/dirs.go b/dirs/dirs.go
index 1d0175eea..fd2057be3 100644
--- a/dirs/dirs.go
+++ b/dirs/dirs.go
@@ -167,7 +167,7 @@ func SetRootDir(rootdir string) {
 	}
 	GlobalRootDir = rootdir
 
-	if release.DistroLike("fedora", "arch") {
+	if release.DistroLike("fedora", "arch", "manjaro") {
 		SnapMountDir = filepath.Join(rootdir, "/var/lib/snapd/snap")
 	} else {
 		SnapMountDir = filepath.Join(rootdir, defaultSnapMountDir)
-- 
2.15.1

