--- fglrx-15.201.1151/archive_files/common/lib/modules/fglrx/build_mod/kcl_wait.c	2015-09-09 00:57:14.000000000 +0200
+++ fglrx-15.201.1151/archive_files/common/lib/modules/fglrx/build_mod/kcl_wait.c	2017-07-16 10:00:45.878601496 +0200
@@ -49,7 +49,7 @@
  */
 KCL_WAIT_Handle ATI_API_CALL KCL_WAIT_Add(KCL_WAIT_ObjectHandle object_handle)
 {
-    wait_queue_t* wait_handle = kmalloc(sizeof(wait_queue_t), GFP_KERNEL);
+    wait_queue_entry_t* wait_handle = kmalloc(sizeof(wait_queue_entry_t), GFP_KERNEL);
 
     if (!wait_handle)
     {
@@ -68,7 +68,7 @@ KCL_WAIT_Handle ATI_API_CALL KCL_WAIT_Ad
  */
 KCL_WAIT_Handle ATI_API_CALL KCL_WAIT_Add_Exclusive(KCL_WAIT_ObjectHandle object_handle)
 {
-    wait_queue_t* wait_handle = kmalloc(sizeof(wait_queue_t), GFP_KERNEL);
+    wait_queue_entry_t* wait_handle = kmalloc(sizeof(wait_queue_entry_t), GFP_KERNEL);
 
     if (!wait_handle)
     {
@@ -89,7 +89,7 @@ void ATI_API_CALL KCL_WAIT_Remove(KCL_WA
                                   KCL_WAIT_ObjectHandle object_handle)
 {
     remove_wait_queue((wait_queue_head_t*)object_handle,
-                      (wait_queue_t*)wait_handle);
+                      (wait_queue_entry_t*)wait_handle);
 
     if (wait_handle)
     {
