diff -rupN a/dde-session-ui.pro b/dde-session-ui.pro
--- a/dde-session-ui.pro	2017-12-13 08:43:33.000000000 +0100
+++ b/dde-session-ui.pro	2018-02-04 14:34:53.762051944 +0100
@@ -5,11 +5,8 @@ SUBDIRS += dde-shutdown \
            dde-switchtogreeter \
            dde-lowpower \
            dde-osd \
-#           dde-offline-upgrader \
            dde-suspend-dialog \
            dde-warning-dialog \
-           dde-welcome \
-           dde-wm-chooser \
            dmemory-warning-dialog
 
 include(common.pri)
